react-ui/src/components/Reviewer/Questions.js

```diff
@@ -10,39 +10,27 @@ class Questions extends Component {
   constructor(props) {
     super(props);
     this.handleSubmit = this.handleSubmit.bind(this);
-    this.updateAnswer = this.updateAnswer.bind(this);
-    this.handleChange = this.handleChange.bind(this);
+    this.handleInputChange = this.handleInputChange.bind(this);
     this.state = {
       answer: '',
       submitResponse: false,
       questionsLeft: 0, // TODO need some way to stop at the last question and go to the final page
     };
   }
 
-  updateAnswer(e) {
-    e.preventDefault;
-    const answer = this.refs.input ? this.refs.input.value : this.state.answer;
-    this.setState({
-      answer: answer,
-    })
```

Comment:
This was one of two places where we were updating the state for answer. Seemed problematic so I went with the renamed `handleInputChange` method. I also wasn't sure what the `ref.input` where for, so let me know if I'm missing something important.

Reply:
I just hadn't noticed that things were named differently. I was trying everything I could find to get the input.